content/metadata.yaml

```
@@ -278,6 +313,12 @@ authors:
           - Software
           - Project Administration
           - Writing - Review & Editing
+      pharmaceuticals:
+        order: 10
+        contributions:
+          - Writing - Review & Editing
+          - Project Administration
+          - Visualization
```

```suggestion
          - Visualization
          - Software
```
Adding software because this is the section where the external resources Python scripts are used.